src/Symfony/Component/Workflow/MarkingStore/MultipleStateMarkingStore.php

```diff
@@ -16,17 +16,17 @@
 use Symfony\Component\Workflow\Marking;
 
 /**
- * PropertyAccessorMarkingStore.
+ * MultipleStateMarkingStore.
```

Comment:
You should describe the class instead of just using the class name, which does not really help.